namer/namer.cc

```diff
@@ -130,13 +130,18 @@ struct FoundMethod final {
 };
 
 struct Modifier {
-    DefinitionKind kind;
+    enum : u1 {
+        Class = 0,
+        Method = 1,
+        ClassOrStaticField = 2,
+    } kind;
```

Comment:
```suggestion
    };
    Kind kind;
```

i prefer no anonymous enums, so that you can write them in type signatures if you need to.